sources/forms-docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+services:
+  mongo:
+    image: mongo:4.1
+    restart: always
+    volumes:
+      - mdb-data:/data/db
+    environment:
+      MONGO_INITDB_ROOT_USERNAME:
+      MONGO_INITDB_ROOT_PASSWORD:
+  
+  formio:
+    build:
+      context: ./packages/forms
+      args:
+        - FORMIO_SOURCE_REPO_URL=${FORMIO_SOURCE_REPO_URL}
+        - FORMIO_SOURCE_REPO_TAG=${FORMIO_SOURCE_REPO_TAG}
+    # The app will restart until Mongo is listening
+    restart: always
+    links:
+      - mongo
+    ports:
+      - "3001:3001"
+    # The application wants to download things to the local directory
+    volumes:
+      - ./:/app:rw
+    environment:
+      DEBUG: formio:*
+      NODE_CONFIG: '{"mongo": "mongodb://mongo:27017/formio"}'
+      ROOT_EMAIL: admin@example.com
+      ROOT_PASSWORD: CHANGEME
+    stdin_open: true  # -i
+    tty: true         # -t
+
```

Comment:
I believe that we need to make the network here explicit and with the same name as `docker-compose.yml`, as shown below (`camunda-docker-compose-core.yml` also needs to be changed). Also, I believe that the env `FORMS_URL=${FORMS_URL}` on API docker-compose must be changed to `FORMS_URL=formio:3001`, the same that @dheepak-aot did for Redis and kindly adjusted to Zeebe also.
```yaml
networks:
  local-network:
    driver: bridge
```